### PR TITLE
added endpoints, serializers.py, urls.py and views.py

### PR DESCRIPTION
**Pull request by Afformativ**

### CODE REVIEW

File: backend/wine_map/views.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from rest_framework import generics
+from .models import Wine
+from rest_framework.pagination import LimitOffsetPagination
+from .serializers import WineSerializer
```
Comment (antos07) [inline backend/wine_map/views.py:4]: According to [PEP-8](https://peps.python.org/pep-0008/#imports:~:text=Imports%20should%20be,group%20of%20imports.) you should split your imports into three groups (from standard library, third-party imports and local app specific imports) with a blank line between each

```suggestion
from rest_framework import generics
from rest_framework.pagination import LimitOffsetPagination

from .models import Wine
from .serializers import WineSerializer
```

File: backend/wine_map/serializers.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from rest_framework import serializers
+from .models import Wine, Brand, Country, Sweetness, Taste, DishCategory
```
Comment (antos07) [inline backend/wine_map/serializers.py:2]: For the same reason as above

```suggestion

from .models import Wine, Brand, Country, Sweetness, Taste, DishCategory
```

**Review by antos07 — COMMENTED** [review]
Some non-essential code-style violations

File: backend/wine_map/views.py (hunk 1 above)
Comment (Nattalli) [reply backend/wine_map/views.py:4]: you can use combination `control + option + O`

**Review by Nattalli — CHANGES_REQUESTED** [review]
why didn't you register your urls here https://github.com/Nattalli/lteam-wine-map/blob/main/backend/config/urls.py
now it willn't work

**Review by Nattalli — APPROVED** [review]

**Review by antos07 — APPROVED** [review]

**Pull request merged**